client/components/account-balances/index.tsx

```diff
@@ -2,50 +2,24 @@
  * External dependencies
  */
 import React from 'react';
-import { Card, CardHeader, Flex, TabPanel } from '@wordpress/components';
-
+import { Card, CardHeader } from '@wordpress/components';
 /**
  * Internal dependencies
  */
 import './style.scss';
+import AccountBalancesTabPanel from './balances-tab-panel';
 
 /**
  * Renders an overview of the account's balances.
  *
- * @return {JSX.Element} Rendered element with the account balances card.
+ * @return {React.FunctionComponent} Rendered element with the account balances card.
```

Comment:
I was going to comment on this return type in JSDoc and then I got rabbitholed on the whole debate.

`React.FC/FunctionComponent` OR `JSX.Element` seems fine unless there is a specific need.

Also, this is JSDoc, so it makes no difference to TS.

Reply:
I don't recall actively changing this. 🤔 I wonder if my linter did that automatically. I'll change it back because I use `JSX.Element` in other places and just contributing to the inconsistency of it all. 😅

Reply:
I originally had this as `React.FunctionComponent` in the branch you've based this off, so maybe it reverted back during a merge.